transforms/ember-object/__testfixtures__/options/decorators-disabled.input.js

```diff
@@ -4,74 +4,74 @@ import { inject as service } from "@ember/service";
 import { on } from "@ember/object/evented";
 
 // Do not transform
-const Foo = EmberObject.extend({
+const Foo1 = EmberObject.extend({
   computed: computed({
     get() {},
     set() {}
   })
 });
 
 // Do not transform
-const Foo = EmberObject.extend({
+const Foo2 = EmberObject.extend({
   statefulObject: {},
   statefulArray: []
 });
 
 // Do not transform
-const Foo = EmberObject.extend({
+const Foo3 = EmberObject.extend({
   tagName: "div"
 });
 
 // Do not transform
-const Foo = EmberObject.extend({
+const Foo4 = EmberObject.extend({
   classNames: []
 });
 
 // Do not transform
-const Foo = EmberObject.extend({
+const Foo5 = EmberObject.extend({
   classNameBindings: ["foo"],
```

Comment:
I kind of wish we'd have created a transitional transform for this like:

```ts
class Foo5 extends EmberObject {
  @concat(["foo"]) classNameBindings;
}
```

which would have created a getter like:

```ts
class Foo5 extends EmberObject {
  get classNameBindings() {
    return [...super.classNameBindings, "foo"];
  }
}
```

Oh well!

Reply:
There is a `@classNameBindings` class decorator tested in some of the other tests. It's just disabled for this one.